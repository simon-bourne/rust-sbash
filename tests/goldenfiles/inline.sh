inline-fn () { 
    echo inline-fn
};