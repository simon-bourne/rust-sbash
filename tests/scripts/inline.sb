inline fn test-fn() {
    echo inline-fn
}
