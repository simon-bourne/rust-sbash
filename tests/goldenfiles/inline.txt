test-fn () { 
    echo inline-fn
};